test/e2e-tests/template.config.json

```diff
@@ -0,0 +1,37 @@
+{
+  "namespace": "my-test-namespace",
+  "image_registry_username": "username+myrobot",
+  "image_registry_password": "",
+  "target_image_tags": ["quay.io/username/repo repo:latest"],
```

Comment:
Is this tag correct? It should either be `["quay.io/username/repo", "repo:latest"]` or `["quay.io/username/repo:latest"]`, right?

Reply:
Yep it was the first one